fv3config/_config.py

```diff
@@ -99,21 +92,11 @@ def write_run_directory(config, target_directory):
     check_if_data_is_downloaded()
     input_directory = os.path.join(target_directory, 'INPUT')
     restart_directory = os.path.join(target_directory, 'RESTART')
-    initial_conditions_dir = get_initial_conditions_directory(config)
-    base_forcing_dir = get_base_forcing_directory(config)
-    orographic_forcing_dir = get_orographic_forcing_directory(config)
-    field_table_filename = get_field_table_filename(config)
-    diag_table_filename = get_diag_table_filename(config)
-    data_table_filename = get_data_table_filename(config)
+    asset_list = config_to_asset_list(config)
     for directory in [target_directory, input_directory, restart_directory]:
         if not os.path.isdir(directory):
             os.mkdir(directory)
```

Comment:
Can this logic be moved into `write_asset_list`? It seems a little low-level for this top-level routine.

Reply:
In fact, isn't this code unneeded since this line is in `write_asset`?
```
        os.makedirs(os.path.dirname(target_path))
```

Reply:
Thanks yeah I just realized this last night looking the code. Used to be needed, but no longer!